src/js/events/MarathonActions.js

```diff
@@ -89,9 +89,15 @@ module.exports = {
     });
   },
 
-  editService: function (data) {
+  editService: function (data, force) {
+    let url = `${Config.rootUrl}${Config.marathonAPIPrefix}/apps/${data.id}`;
+
+    if (force === true) {
+      url += '?force=true';
```

Comment:
It would be good to add a simple unit test to make sure that the `force` argument changes the requested URL.